includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -1160,6 +1160,10 @@ public function get_formatted_item_price ( $item, $type, $tax_display = '' ) {
 		return $item_price;
 	}
 
+	/**
+	 * Legacy function (v3.7.2 or inferior)
```

Comment:
Can we have `_deprecated_function()` for these legacy functions? And maybe calling the new functions like the `$this->number( $document_type );` instead of their current codes?